packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -1023,6 +1030,11 @@ export class LocalDetachedFluidDataStoreContext
 
         super.bindRuntime(dataStoreChannel);
 
+        // Load the handle to initialize the object. The only implementation of IFluidDataStoreChannel is
+        // FluidDataStoreRuntime, which exposes its handle.
+        const channelAsFluidObject: FluidObject<IFluidHandle> = (dataStoreChannel as any);
+        await channelAsFluidObject.IFluidHandle?.get();
+
```

Comment:
I brought the two changes to this file from Tony's earlier draft of this change. I don't think I would have been able to realize if they were necessary without someone pointing them out. If someone can clarify the reasoning why we'd want to "force" initialization of the handle here, it'd be much appreciated.

Reply:
I do not think runtime should do it. So, I'm curious too :)
Unless there is a very good reason, I'd say this is anti-pattern.

Reply:
i believe it has to do with the change in flow. previous after create the only option was to request, which would realize the internal object. in the new flow, we get the handle directly, and never need to call get, as serializing won't realize the inter object. So this code was there to ensure that all new objects are realized